tst/SdpApiTest.cpp

```diff
@@ -411,8 +416,8 @@ TEST_F(SdpApiTest, populateSingleMediaSection_TestTxSendRecv)
 
     // check nack and nack pli lines
     // We assume here DEFAULT_PAYLOAD_H264 is 125 so we know what our offer will generate.
```

Comment:
Don't think this comment is needed anymore with this new change.

Reply:
Done